packages/perseus-editor/src/components/graph-locked-figures/locked-function-settings.tsx

```diff
@@ -200,6 +256,43 @@ const LockedFunctionSettings = (props: Props) => {
     );
 };
 
+type ItemProps = {
+    category: string;
+    example: string;
+    index: number;
+    pasteEquationFn: (property: string, newValue: string) => void;
+};
+
+const ExampleItem = (props: ItemProps): React.ReactElement => {
+    const {category, example, index, pasteEquationFn} = props;
+    const exampleId = useId();
+
+    return (
+        <li key={`${category}-${index}`} className={css(styles.exampleRow)}>
+            <IconButton
+                icon={autoPasteIcon}
+                aria-label="paste example"
+                aria-describedby={exampleId}
+                onClick={() => pasteEquationFn("equation", example)}
+                size="medium"
+                style={styles.copyPasteButton}
+            />
```

Comment:
Optional suggestion: Since content authors usually use desktop + mouse/keyboard, I think it would be appropriate to use a `title` prop here so that someone can hover over the icon to learn what it does.

Reply:
Since `<IconButton>` doesn't have a `title` property available, I'm unable to add this directly. So, leaving this off for now.